generators/client/templates/vue/src/main/webapp/app/account/login.service.ts.ejs

```diff
@@ -12,7 +12,15 @@ export default class LoginService {
   <%_ } else { _%>
   public login(loc = window.location) {
     const port = (loc.port ? ':' + loc.port : '');
-    loc.href = '//' + loc.hostname + port + loc.pathname + 'authorize';
+    let contextPath = location.pathname;
+    if (contextPath.endsWith('accessdenied')) {
+      contextPath = contextPath.substring(0, contextPath.indexOf('accessdenied'));
+    }
+    if (!contextPath.endsWith('/')) {
+      contextPath = contextPath + '/';
+    }
+
+    loc.href = `//${loc.hostname}${port}${contextPath}oauth2/authorization/oidc`;
```

Comment:
We should add some documentation here as we have as a comment in the main generator:

```
// If you have configured multiple OIDC providers, then, you can update this URL to /login.
// It will show a Spring Security generated login page with links to configured OIDC providers.
```